src/Symfony/Component/ClassLoader/Tests/Fixtures/Namespaced/WithStrictTypes.php

```diff
@@ -4,7 +4,7 @@
  * foo
  */
 
-declare (strict_types = 1);
+declare(strict_types=1);
```

Comment:
because it uses outdated `braces` rule version which isn't really ready for php7